src/main/common/utils.h

```diff
@@ -96,15 +96,21 @@
         const typeof( ((type *)0)->member ) *__mptr = (ptr);    \
         (type *)( (char *)__mptr - offsetof(type,member) );}))
 
-static inline int16_t cmp16(uint16_t a, uint16_t b) { return (int16_t)(a-b); }
-static inline int32_t cmp32(uint32_t a, uint32_t b) { return (int32_t)(a-b); }
+static inline int16_t cmp16(uint16_t a, uint16_t b) {
+    return (int16_t)(a - b);
+}
+static inline int32_t cmp32(uint32_t a, uint32_t b) {
+    return (int32_t)(a - b);
+}
 
 // using memcpy_fn will force memcpy function call, instead of inlining it. In most cases function call takes fewer instructions
 //  than inlined version (inlining is cheaper for very small moves < 8 bytes / 2 store instructions)
 #if defined(UNIT_TEST) || defined(SIMULATOR_BUILD)
 // Call memcpy when building unittest - this is easier that asm symbol name mangling (symbols start with _underscore on win32)
 #include <string.h>
-static inline void  memcpy_fn ( void * destination, const void * source, size_t num ) { memcpy(destination, source, num); }
+static inline void  memcpy_fn ( void * destination, const void * source, size_t num ) {
+    memcpy(destination, source, num);
```

Comment:
Codacy found an issue: [Does not check for buffer overflows when copying to destination (CWE-120).  Make sure destination can always hold the source data. ](https://app.codacy.com/gh/emuflight/EmuFlight/pullRequest?prid=6326343)